consensus/config/src/committee.rs

```diff
@@ -132,7 +130,7 @@ impl Committee {
 ///
 /// NOTE: this is intentionally un-cloneable, to encourage only copying relevant fields.
 /// AuthorityIndex should be used to reference an authority instead.
-#[derive(Serialize, Deserialize)]
+#[derive(Clone, Serialize, Deserialize)]
```

Comment:
Why is it needed to clone `Authority`, which can be expensive? Why not pass `AuthorityIndex` or `&Authority`?